sec1_intro_and_setup/test_app/config/initializers/filter_parameter_logging.rb

```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Whats this file for?

Reply:
This file allows you to put sensitive data that you don't want to it to be shown in log.

Reply:
Huh? What you mean this file allows to put sensitive data? put where?